### DISCUSSION TITLE
Trying own all badges of GtHuB

### DISCUSSION THREAD

**Discussion by bor-ges** [community/community#134666] [Pull Requests]
### Select Topic Area

Question

### Body

I want to earn all types off badges on github so if anyone can help me I'm gonna fell very hapy!!!

**Comment by bor-ges** [comment]
teste

**Reply by bor-ges** [reply]
yes

**Comment by bor-ges** [comment]
another test

**Reply by bor-ges** [reply]
try again

**Comment by bor-ges** [comment]
test novamente

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/134669#discussioncomment-10246676) for why this discussion was closed and locked.
